exporter/collector/config.go

```diff
@@ -163,6 +163,12 @@ type LogConfig struct {
 	// Defaults to empty, which won't include any additional resource labels.
 	ResourceFilters []ResourceFilter `mapstructure:"resource_filters"`
 	ClientConfig    ClientConfig     `mapstructure:",squash"`
+	// MaxEntrySize defines the maximum size of an individual LogEntry in bytes. If a LogEntry
+	// is larger than this size, it will be split and sent as multiple entries. Default: 256000 (256KB).
+	MaxEntrySize int `mapstructure:"max_entry_size"`
```

Comment:
If we don't expect anyone to set this, should we omit it from the config?  Is it actually neccessary?

Reply:
We don't need it, it just felt weird to have request size configurable without it

Reply:
Oh, I see.  We need to be able to set MaxRequestSize, but not MaxEntrySize?

Reply:
yeah, for the integration test we need to either be able to configure MaxRequestSize (to simulate a smaller limit) or use a fixture with >10MB requests. 

This was the easiest way I could find to pass a request size, but we could do it in ways that don't affect the collector config. Either way though, it's going to involve exporting something around this request size publicly since it's in a different package than the integration test